packages/backend/src/projectAdapters/projectAdapter.ts

```diff
@@ -78,6 +79,18 @@ export const projectAdapterFromConfig = async (
                 config.authorization_method === 'installation_id'
                     ? await getInstallationToken(config.installation_id)
                     : config.personal_access_token;
+            if (githubToken === undefined) {
+                throw new ParameterError(
```

Comment:
This should also help debugging https://github.com/lightdash/lightdash/issues/13899 if it happens again